packages/craco-config-flex-plugin/jest.config.js

```diff
@@ -0,0 +1,12 @@
+const base = require('./../../jest.config.base');
+const pkg = require('./package');
+
+module.exports = {
+  ...base,
+  name: pkg.name,
+  displayName: pkg.name,
+  rootDir: '../..',
+  testMatch: [
+    `<rootDir>/packages/${pkg.name}/**/*.test.ts`
```

Comment:
.ts or .tsx?

Reply:
Plugin builder doesn't have any react components; also these is just copy/paste from the other jest.config files.